src/components/Block/View.jsx

```diff
@@ -125,10 +161,29 @@ const View = (props) => {
             />
           </Form.Field>
         )}
+        {includes(modules, 'vimeo') && (
+          <Form.Field>
+            <Checkbox
+              toggle
+              label="Vimeo"
+              onChange={() => setConfirmVimeo(!confirmVimeo)}
+              checked={confirmVimeo}
+            />
+          </Form.Field>
+        )}
+        {includes(modules, 'twitter') && (
+          <Form.Field>
+            <Checkbox
+              toggle
+              label="Twitter"
+              onChange={() => setConfirmTwitter(!confirmTwitter)}
+              checked={confirmTwitter}
+            />
+          </Form.Field>
+        )}
```

Comment:
@Molochem This PR appears to also include the changes from https://github.com/kitconcept/volto-dsgvo-banner/pull/23

Ideally, if you start a new PR, you should go back to the main branch before creating a new branch, so that we aren't mixing multiple changes together until they are merged. Your colleagues can show you the details if you aren't sure how.

I'm okay with keeping the changes here this time, but then you have to make sure the `news` entry from #23 is also included.